compiler-plugin-tests/src/test/resources/diagnostics/sample11/main.bal

```diff
@@ -47,4 +48,6 @@ public function main() returns error? {
     xml x1 = xml `<foo><bar>2</bar><car></car></foo>`;
     Foo actual = check xmldata:fromXml(x1);
     Bar result = check xmldata:fromXml(x1);
+    io:println(actual);
+    io:println(result);
```

Comment:
Do we need to print the outputs here?

Reply:
No need but I added it as otherwise this would also give a warning.